docs/sources/v0.47.x/testing-guides/injecting-faults-with-xk6-disruptor/first-steps.md

```diff
@@ -2,30 +2,212 @@
 title: 'xk6-disruptor first steps'
 excerpt: 'xk6-disruptor is a k6 extension providing fault injection capabilities to k6.'
 weight: 01
+aliases:
+  - /docs/k6/latest/javascript-api/xk6-disruptor/get-started/ 
 ---
 
-# xk6-disruptor first steps
+# First steps
 
-[xk6-disruptor](https://github.com/grafana/xk6-disruptor) is an extension that adds fault injection capabilities to k6.
+This document will guide you through running your first fault injection test using xk6-disruptor, while providing pointers to the more detailed documentation sections along the way. We will cover the following sections:
 
-It provides a Javascript [API](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/xk6-disruptor/) to inject [faults](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/xk6-disruptor/faults) such as errors and delays into HTTP and gRPC requests served by selected Kubernetes [Pods](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/xk6-disruptor/poddisruptor) or [Services](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/xk6-disruptor/servicedisruptor).
+1. [Checking requirements](#checking-requirements)
+1. [Installing xk6-disruptor](#installing-xk6-disruptor)
+1. [Creating a simple k6 test](#creating-a-simple-k6-test)
+1. [Add fault injeciton capabilities](#add-fault-injection-capabilities)
 
+# Checking requirements
+
+xk6-disruptor focuses on injecting faults on applications running in Kubernetes. If you already have your application deployed to a development Kubernetes cluster, you are all set! If you don't, we will guide you through deploying a demo application on a local `minikube` cluster.
```

Comment:
I was thinking the opposite, actually: I added the example so people can still follow if they don't have their application deployed and accessible right when they are reading the article, but my original intention was for the guide to be the point of reference for developers/sres who do have an application and want to start using the disruptor. I think this was the feedback we got and originated this page: "I have an application but the multi-paged structure is hard to follow end to end to get me from zero to something"

Reply:
> I added the example so people can still follow if they don't have their application deployed and accessible right when they are reading the article
> think this was the feedback we got and originated this page: "I have an application but the multi-paged structure is hard to follow end to end to get me from zero to something"

That is still the goal and I understand the intention,  but my own experience reading the content was the opposite.
It was unclear what the general steps are and what is specific to the demo. And the example makes a very long chunk of text and code in the middle of the guide!

The main problem, I see is that any example with the disruptor API would be application specific.

I still think it would be better to put all the steps together and then demo them in an example. In this way, if someone already has an application the demo won't be in the middle.